webapp/channels/src/components/file_preview/file_preview.tsx

```diff
@@ -66,19 +66,15 @@ export default class FilePreview extends React.PureComponent<Props> {
                     imageClassName += ' normal';
                 }
 
-                let thumbnailUrl = getFileThumbnailUrl(info.id);
-                if (Utils.isGIFImage(info.extension) && !info.has_preview_image) {
```

Comment:
We should still respect `has_image_preview` here and have the image URL use either `getFileThumbnailUrl` or `getFileUrl`, but we could probably get rid of the check for the image type here

Reply:
okay will do this change and update it in the next commit